client/src/main/scala/com.gu.contentapi.client/ContentApiBackoff.scala

```diff
@@ -30,10 +30,10 @@ abstract class ContentApiBackoff(implicit executor: ScheduledExecutor) extends P
 
   def currentState: ContentApiBackoff = self
 
-  def execute(operation: => Future[HttpResponse])(implicit context: ExecutionContext): Future[HttpResponse] = {
+  def execute(operation: Int => Future[HttpResponse])(implicit context: ExecutionContext): Future[HttpResponse] = {
```

Comment:
This breaking API change is associated with a minor upgrade? Is this ok? i.e. do we not adhere to semver?

Reply:
I think execute is not directly called from outside - I'll re-check. But actually, maybe a major version bump is the way to go for this anyway.

Reply:
So `backoffStrategy.execute` is called from within `private def fetchResponse` so I think we're protecting consumers of the library from that change. I'll bump to 16.0 anyway though I think.